TeamCode/src/main/java/org/firstinspires/ftc/teamcode/drive/SampleMecanumDrive.java

```diff
@@ -394,4 +394,28 @@ public void setMotorPowers(double v, double v1, double v2, double v3) {
     public double getRawExternalHeading() {
         return imu.getAngularOrientation().firstAngle;
     }
+
+    @Override
+    public Double getExternalHeadingVelocity() {
+        // TODO: This must be changed to match your configuration
+        //                           | Z axis
+        //                           |
+        //     (Motor Port Side)     |   / X axis
+        //                       ____|__/____
+        //          Y axis     / *   | /    /|   (IO Side)
+        //          _________ /______|/    //      I2C
+        //                   /___________ //     Digital
+        //                  |____________|/      Analog
+        //
+        //                 (Servo Port Side)
+        //
+        // In this diagram, the side normal to the x axis facing away from you is the side with the
+        // USB port(s)
```

Comment:
Is it clearer to say "the positive x axis points toward the USB port(s)"?

Reply:
Would that your statement verbatim be more clear?

Reply:
Yes, that phrasing suffices verbatim.